Zend/tests/clone/clone_with_003.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Clone with supports property hooks
+--FILE--
+<?php
+
+class Clazz {
+	public string $hooked = 'default' {
+		set {
+			$this->hooked = strtoupper($value);
+		}
+	}
+}
+
+$c = new Clazz();
+
+var_dump(clone($c, [ 'hooked' => 'updated' ]));
```

Comment:
Nit: We don't usually add spaces in `()` or `[]`.

Reply:
This is already wildly inconsistent in tests and I'd like to avoid test churn at this point in the PR. I'll try to keep it in mind for the future, though.